gc/base/MemoryPoolLargeObjects.hpp

```diff
@@ -115,9 +117,42 @@ class MM_MemoryPoolLargeObjects : public MM_MemoryPool {
 	double resetTargetLOARatio(MM_EnvironmentBase*);
 	void resetLOASize(MM_EnvironmentBase*, double newLOARatio);
 	void redistributeFreeMemory(MM_EnvironmentBase*, uintptr_t newOldAreaSize);
-	bool isSizeEnoughForLOA(MM_EnvironmentBase* env, uintptr_t newOldAreaSize)
+
+	/* check if new LOA size is enough for largeObjectMinimumSize and set _loaSize, _soaSize, _currentLOARatio and _currentLOABase */
+	bool isSizeEnoughForLOA(MM_EnvironmentBase* env, uintptr_t newLOASize, double newLOARatio, void* newLOABase=NULL)
```

Comment:
This function has many side effects, and to me `isSizeEnoughForLOA` implies that it just does a simple check.  I don't have any great suggestions, but can we try to find a better name for this function? Maybe `resizeLOA()`?

Reply:
This file doesn't put types on their own line. Not sure if we want to fix that in this pr though.
```suggestion
	MMINLINE bool
	isSizeEnoughForLOA(MM_EnvironmentBase* env, uintptr_t newLOASize, double newLOARatio, void* newLOABase=NULL)
```

Reply:
rename to checkAndSetSizeForLOA()